app/webpacker/lib/utils/competition-table.js

```diff
@@ -89,13 +89,12 @@ export function timeDifferenceBefore(competition, refDate) {
 }
 
 export function numberOfDaysAfter(competition, refDate) {
-  const parsedStartDate = parseDateString(competition.end_date).endOf('day');
-  const parsedRefDate = DateTime.fromISO(refDate);
+  const parsedEndDate = parseDateString(competition.end_date).startOf('day');
+  const parsedRefDate = DateTime.fromISO(refDate, { zone: 'utc' }).startOf('day');
 
-  const numberOfDays = parsedStartDate.diff(parsedRefDate, 'days').days;
+  const numberOfDays = parsedRefDate.diff(parsedEndDate, 'days').days;
 
-  // Floor is used here because we want to show 0 days after the competition if it's the same day
-  return Math.floor(Math.abs(numberOfDays));
+  return Math.max(0, numberOfDays);
```

Comment:
Why `max` instead of something like `abs`? I read this as "make sure it's not negative", so what are the circumstances that can even make a negative date appear?